src/ui_elements/menu_character.gd

```diff
@@ -0,0 +1,21 @@
+extends Control
+
+onready var skeleton = $Viewport/Skeleton
+
+# --Public Variables--
+
+# Set the outline shader color
+func setOutline(color: Color) -> void:
+	$CharacterTexture.material.set_shader_param("line_color", color)
+
+# Apply config from the appearance public variables to skeleton
+func applyFromAppearance():
+	skeleton.applyAppearance(Appearance.currentOutfit, Appearance.currentColors)
+
+# Set config to skeleton from inputted variables
+func setAppearance(outfit: Dictionary, colors: Dictionary):
+	skeleton.applyAppearance(outfit, colors)
+
```

Comment:
Put an assert either here (assert len(outfit)>0 or something, I don't know how "isempty" work in gdscript), so in the future we'll find it if something is doing a f*cky-wacky.